Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/messagehandler/rediscache.properties

```diff
@@ -0,0 +1,20 @@
+DIALOG_TITLE=New Redis Cache
```

Comment:
Is these properties ui related or core logic? if ui related, don't put into azuretoos-core